lib/fizzy/execute.cpp

```diff
@@ -517,6 +517,30 @@ __attribute__((no_sanitize("float-divide-by-zero"))) inline constexpr T fdiv(T a
     return a / b;  // For IEC 559 (IEEE 754) floating-point types division by 0 is defined.
 }
 
+template <typename T>
+inline constexpr T fmin(T a, T b) noexcept
+{
+    if (std::isnan(a) || std::isnan(b))
+        return std::numeric_limits<T>::quiet_NaN();
```

Comment:
Is it guaranteed that it's a canonical NaN? I know we have a test for that, but maybe it's worth a `static_assert`?

Reply:
No. But I don't have a way to check it here.